libs/langchain-community/src/callbacks/handlers/lunary.ts

```diff
@@ -0,0 +1 @@
+export { LunaryHandler } from "lunary/langchain";
```

Comment:
This is fine but just keep in mind we won't be able to update if refactors happen